src/main/resources/assets/admin/common/js/ui/dialog/multistep/MultiStepDialog.ts

```diff
@@ -0,0 +1,281 @@
+import {DialogStep} from './DialogStep';
+import {Element} from '../../../dom/Element';
+import * as Q from 'q';
+import {ModalDialog, ModalDialogConfig} from '../ModalDialog';
+import {ActionButton} from '../../button/ActionButton';
+import {DivEl} from '../../../dom/DivEl';
+import {i18n} from '../../../util/Messages';
+import {Action} from '../../Action';
+import {AppHelper} from '../../../util/AppHelper';
+import {DefaultErrorHandler} from '../../../DefaultErrorHandler';
+
+
+export interface MultiStepDialogConfig
+    extends ModalDialogConfig {
+
+    steps: DialogStep[];
+
+    submitCallback?: { (): void; };
```

Comment:
Why is this an object? Should be `submitCallback?: () => void`

Reply:
Updated